app/controllers/shelters_controller.rb

```diff
@@ -10,12 +10,16 @@ def new
   end
 
   def create
-    shelter = Shelter.create!(shelter_params)
-    redirect_to '/shelters'
-    # if shelter.save  == true #boolean response, true if saved
-    #   redirect_to '/shelters'
-    # else
-    #   render.newse
+
+    shelter = Shelter.create(shelter_params)
+
+    if shelter.save
+      flash[:notice] = "You have created #{shelter.name}"
+      redirect_to '/shelters'
+    else
+      flash[:notice] = "You must fill out all fields"
```

Comment:
had trouble getting a list of fields that were left blank. It may have to be a private method in shelter_controller